modelgauge/main.py

```diff
@@ -228,6 +241,81 @@ def run_test(
     print("Full TestRecord json written to", output_file)
 
 
+@modelgauge_cli.command()
+@click.option(
+    "sut_names",
```

Comment:
This is nit-picky buy I think `sut_uids` would be more accurate

Reply:
Excellent nit-pick! Fixed.

Reply:
Thanks!